saged/tissue_split.py

```diff
@@ -0,0 +1,230 @@
+"""
+This experiment determines whether models pretrained on data from different tissues than they
+are predicting do better than models without pretraining
+"""
+import argparse
+import copy
+import gc
+import random
+
+import sklearn.metrics
+import torch
+
+import yaml
+
+from saged import utils, datasets, models
+from saged.models import LogisticRegression, PytorchSupervised
+
+
+PREDICT_TISSUES = ['Blood', 'Breast', 'Stem Cell', 'Cervix', 'Brain', 'Kidney',
```

Comment:
Seems like these `PREDICT_TISSUES` are the same across all of your tissue pretraining experiments - maybe you could put this list somewhere centralized and import/load it in each script? Unless you want to set it individually in each script.

Reply:
Yeah, that's a good point, I'll save that change for the refactoring PR though